descriptors/ModuleDescriptor-template.json

```diff
@@ -129,6 +120,20 @@
             "change-manager.jobexecutions.post"
           ]
         },
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/data-import/uploadDefinitions/{uploadDefinitionId}/files/{fileId}/assembleStorageFile",
+          "permissionsRequired": [
+            "data-import.assembleStorageFile.post"
+          ],
+          "modulePermissions": [
+            "configuration.entries.collection.get",
+            "change-manager.jobexecutions.post",
```

Comment:
could you say please do we need to provide this permission (necessary for jobExecution creation) for this endpoint?

Reply:
good catch, thanks!

Reply:
We don't need the `post` one, good catch, thanks!

Reply:
understood, thanks! "configuration.entries.collection.get" seems is not needed as well, right?

Reply:
@RuslanLavrov fixed!

Reply:
thanks!